lighting/light/falloff.glsl

```diff
@@ -1,10 +1,20 @@
-#include "../../math/saturate.glsl"
+#include "../../math/const.glsl"
+
+/*
+contributors:  Shadi El Hajj
+description: Light Falloff equation, based on the model in Brian Karis' paper "Real Shading in Unreal Engine 4"
+use: float falloff(float distance, float lightRadius)
+license: MIT License (MIT) Copyright (c) 2024 Shadi EL Hajj
+*/
 
 #ifndef FNC_LIGHT_FALLOFF
 #define FNC_LIGHT_FALLOFF
-float falloff(const in float _dist, const in float _lightRadius) {
-    float att = saturate(1.0 - _dist * _dist / (_lightRadius * _lightRadius));
+
+float falloff(float distance, float lightRadius) {
```

Comment:
`distance` is a reserve word and some drivers have problems with it. Suggesting using something like `dist`

Reply:
Excellent point. Fixed.